src/Tasks/AxReference.cs

```
@@ -77,7 +77,10 @@ internal bool GenerateWrapper(out ComReferenceWrapperInfo wrapperInfo)
 
             var axImp = new ResolveComReference.AxImp();
 
-            axImp.ActiveXControlName = ReferenceInfo.strippedTypeLibPath;
+            if (ReferenceInfo != null)
```

I would fail here instead of silently skipping the execution.